lib/scrabble.rb

```diff
@@ -1,7 +1,29 @@
 class Scrabble
 
-  def score(word)
-    1
+  attr_reader :word, :scoring
+
+  def initialize(word)
+    @word = word
+    @scoring = []
+  end
```

Comment:
The spec's interaction pattern requests that initializing Scrabble (Scrabble.new) does not take an argument, but that when you call Scrabble.new.score(), it does. Make sure to pay careful attention to the spec so you build out your program as expected.  
For example: 
game = Scrabble.new
game.score("hello") 

or 

Scrabble.new.score("hello")